python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -863,6 +871,13 @@ def run(self, exposure=None, selectSources=None, templateExposure=None, template
         # END (if self.config.doSubtract)
         if self.config.doDetection:
             self.log.info("Running diaSource detection")
+
+            # Rescale difference image variance plane
+            if self.config.doScaleDiffimVariance:
+                self.log.info("Rescaling diffim variance")
+                diffimVarFactor = self.scaleVariance.run(subtractedExposure.getMaskedImage())
+                self.metadata.add("scaleDiffimVarianceFactor", diffimVarFactor)
+
```

Comment:
Please add a log message stating the difference image variance scaling that is being used.